_targets.R

```diff
@@ -0,0 +1,34 @@
+library(targets)
+source('code.R')
```

Comment:
I see you've included your `source()` calls to reference `download_sb_data.R`, `process_model_data.R` and `plot_model_data.R` in the `code.R` file. Instead, these 3 source calls should be included right in `_targets.R`, so that any user of the pipeline can see immediately which scripts are used. Also, when you do make those `source()` calls, no need to provide the full path -- you can simply provide the path relative to the project's root directory, e.g. `source('1_fetch/src/download_sb_data.R')`, rather than `source('~/R/ds-pipelines-targets-1/1_fetch/src/download_sb_data.r')`